.github/workflows/registry-scanner.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Scan Registry
-        uses: boostsecurityio/scanner-registry-action@v1.1.0
+        uses: boostsecurityio/scanner-registry-action@v1.2.0
```

Comment:
It's better to  pin with git commit hash https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions, but I suggest you put the `v1.2.0` in a comment at the end of the line for reference